src/pages/orders/edit-ticket-page.js

```diff
@@ -380,15 +382,17 @@ class EditTicketPage extends React.Component {
                 <Modal show={this.state.showRefundModal} onHide={() => this.setState({showRefundModal:false})} >
                     <Modal.Header closeButton>
                         <Modal.Title>{T.translate("edit_ticket.refund_modal_title")}</Modal.Title>
+                        {`$${entity.net_selling_cost}`}
```

Comment:
this need to have currency symbol and a label informing what it is ( Net Price $ XYZ)